src/Suave.Xsp/Library.fs

```diff
@@ -0,0 +1,178 @@
+﻿namespace Suave
+
+open System
+open System.IO
+open System.Web
+open System.Web.Hosting
+open System.Text
+
+open Suave
+open Suave.Types
+open Suave.Types.Codes
+open Suave.Http
+open Suave.Utils
+
+open Suave.Sockets
+
+module Xsp =
+
+  type Connection(transport : ITransport) =
+    inherit MarshalByRefObject()
```

Comment:
Could you add a comment about why this needs to be MarshalByRef?  I don't see that used so much anymore so am curious.